R/server_manager.R

```diff
@@ -40,6 +40,13 @@ vault_test_server <- function(https = FALSE, init = TRUE,
   vault_server_manager()$new_server(https, init, if_disabled)
 }
 
+vault_exe_filename <- function() {
```

Comment:
please rewrite to take an argument `windows = is_windows()` argument, then test both paths in your tests

Reply:
Done - and I switched to using the existing vault_platform() as the source of truth about what platform we're on.